common/compositor/compositorthread.cpp

```diff
@@ -41,7 +40,9 @@ CompositorThread::~CompositorThread() {
 }
 
 void CompositorThread::Initialize(ResourceManager *resource_manager,
-                                  uint32_t gpu_fd) {
+                                  uint32_t gpu_fd,
+                                  FrameBufferManager *frame_buffer_manager) {
+  fb_manager_ = frame_buffer_manager;
```

Comment:
What happens when the frame data changes after this assignment? Does this assignment account for changes to the frame buffer content instantaneously?

Reply:
fb_manager _ is a pointer, not a reference.